core/src/main/java/io/questdb/griffin/SqlExecutionContext.java

```diff
@@ -79,6 +80,8 @@ default Rnd getAsyncRandom() {
         return getCairoEngine().getMessageBus();
     }
 
+    MicrosecondClock getMicrosecondClock();
```

Comment:
I'd add method `long getMicrosecondTimestamp()` instead so that the clock instance is not cached somewhere

Reply:
changed the method